jurity/recommenders/ctr.py

```diff
@@ -128,22 +183,94 @@ def get_score(self, actual_results: pd.DataFrame, predicted_results: pd.DataFram
             The averaged result(s). The return type is determined by the ``batch_accumulate`` and
             ``return_extended_results`` parameters. See the examples above.
         """
+        n_items = len(actual_results[self._item_id_column].unique())
+
         actual_results = actual_results.set_index([self._user_id_column, self._item_id_column])
         predicted_results = predicted_results.set_index([self._user_id_column, self._item_id_column])
+
         if self.k is not None:
             sorted_clicks = get_sorted_clicks(predicted_results, self._user_id_column, self.click_column, self.k)
         else:
             sorted_clicks = predicted_results
 
-        matches = actual_results.join(sorted_clicks, how='inner', rsuffix='_r')
-        clicks = matches[self.value_column].values
-
-        return self._accumulate_and_return(clicks, batch_accumulate, return_extended_results)
+        if self.estimation == 'matching':
+            return self._get_matching_ctr(actual_results, sorted_clicks, batch_accumulate, return_extended_results)
```

Comment:
is "matching" the same as what's referred to as the Direct Method?

Reply:
(not a PR question tho)

Reply:
"Matching" is not what doubly robust literature refers to as the direct method (which is just using the predicted scores for all items), but Doruk said that a lot of literature on CTR refers to this inner join approach as the direct method. We're calling it matching to disambiguate between the two.

Reply:
Yeah, I see. Thanks for the clarification. My exposure to the literature has also been that matching is what I see papers call direct. Good to make a distinction for ourselves. 

If you are basing the IPS, and DR implementation here on a specific paper, it would be good to add those in the pydoc as a reference.